examples/06_productivity_boosters/01_namedArguments.md

```diff
@@ -1,5 +1,7 @@
 # Named Arguments
 
+As most of programming languages (Java, C++, and other), Kotlin supports passing arguments to methods and constructors according to their order of definition.
```

Comment:
As with most other programming languages (Java, C++, etc.), Kotlin supports passing arguments to methods and constructors according to the order they are defined.